reporting/api/src/main/java/org/jboss/windup/reporting/config/Hint.java

```diff
@@ -284,13 +285,14 @@ private String toString(String title, String text)
         result.append("Hint");
         if (title != null)
             result.append(".titled(\"").append(title).append("\")");
-        result.append(".withText(\"").append(text).append("\")");
         if (effort != 0)
             result.append(".withEffort(").append(effort).append(")");
-        if (links != null && !links.isEmpty())
-            result.append(".with(").append(links).append(")");
         if (tags != null && !tags.isEmpty())
             result.append(".withTags(").append(tags).append(")");
+        if (text != null)
+            result.append("\n\t.withText(\"").append(StringUtils.abbreviate(text.trim(), MAX_LOG_TEXT_LENGTH)).append("\")");
+        if (links != null && !links.isEmpty())
+            result.append("\n\t.with(").append(links).append(")");
```

Comment:
It is better to use `System.lineSeparator()` instead of `\n` to be the same on all OS

Reply:
+1 - System.lineSeparator() is better.

Reply:
It doesn't really matter anymore, all Windows apps can handle \n. But as you like